pkg/admissioncontroller/webhooks/auth/seed/graph/mock/gomock_reflect_045994415/prog.go

```diff
@@ -0,0 +1,64 @@
+package main
```

Comment:
This file shouldn't be here. Please cleanup all temporary artifacts from mockgen.